lib/Platform/Unicode/PlatformUnicodeICU.cpp

```diff
@@ -13,12 +13,21 @@
 
 #include <time.h>
 
+#include <memory>
+
 namespace hermes {
 namespace platform_unicode {
 
-int localeCompare(
-    llvh::ArrayRef<char16_t> left,
-    llvh::ArrayRef<char16_t> right) {
+namespace {
+struct UCollatorDeleter {
+  void operator()(UCollator *coll) {
+    if (coll) {
```

Comment:
I don't believe the deleter is invoked if the pointer is null.

Reply:
Fixed. Thanks, re-read cppreference to make sure.